GTG/core/saved_searches.py

```diff
@@ -160,23 +165,10 @@ def to_xml(self) -> Element:
         return root
 
 
-    def new(self, name: str, query: str, parent: UUID = None) -> SavedSearch:
```

Comment:
This method is still used by the frontend: https://github.com/getting-things-gnome/gtg/blob/master/GTG/gtk/browser/main_window.py#L496

I think we should keep this for now and start refactoring once the tests are actually green.

Reply:
Because it does 2 things that are already done elsewhere: it creates a SavedSearch (which is the responsibility of the SavedSearch class) and it adds it to the store (which is what the `add()` method already does).

So I'd rather not have the duplication.

> I think we should keep this for now and start refactoring once the tests are actually green.

That's fair enough. This is not strictly related to what you're doing in this PR, so we can do it separately.

Reply:
I have reverted it partially and kept it for later reference: https://github.com/getting-things-gnome/gtg/pull/1036/commits/8d8ffcdf3b82c207947163c87a14c55ad4467836